src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -47,27 +47,9 @@ public net.minecraft.client.renderer.RenderType create(Ljava/lang/String;Lcom/mo
 public net.minecraft.client.renderer.RenderType create(Ljava/lang/String;Lcom/mojang/blaze3d/vertex/VertexFormat;Lcom/mojang/blaze3d/vertex/VertexFormat$Mode;IZZLnet/minecraft/client/renderer/RenderType$CompositeState;)Lnet/minecraft/client/renderer/RenderType$CompositeRenderType; # create
```

Comment:
It would be good to add a comment to the top of this file stating that repetitive ATs can be generated using a rule in the `build.gradle` file.